src/style.css

```diff
@@ -1,3 +1,89 @@
+html {
+  --text-height: 15px;
+  --text-gap: 20px;
+  --img-width: 40px;
+  --img-height: 40px;
+  --header-height: 60px;
+  --header-main-text-color: #000;
+  --header-secondary-text-color: #00acdc;
+}
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+.nav_list {
+  margin: 0 20px 0 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  font-style: normal;
+}
+
+.header-img {
+  height: var(--img-height);
+  width: var(--img-width);
+}
+
+.nav_links {
+  display: flex;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav_link {
+  position: relative;
```

Comment:
positioning according to your link is required only for 1 of the elements. Move it to your modifier